### ISSUE TITLE
problem in the output

### ISSUE THREAD

**Issue by rmmal** [argman/EAST#31]
I have trained a model with this command:

python multigpu_train.py --gpu_list=0,1,2 --input_size=512 --batch_size_per_gpu=14 --checkpoint_path=/backup/EAST/ \
--text_scale=1024 --training_data_path=/DATA/EAST/data/ --geometry=RBOX --learning_rate=0.0001 --num_readers=12

and i've waited till:

Step 007130, model loss 0.0316, total loss 0.0827, 7.33 seconds/step, 5.73 examples/second

first Question should i make him , do more iterations or this is enough ???

second Question:
The output of all the images seems to be 1 size , why this is happening ?
i couldn't see many variations in the output dimensions

examples:
![screenshot from 2017-09-11 09-42-23](https://user-images.githubusercontent.com/6699411/30263652-1f2f0344-96d6-11e7-83b4-44ebc74edcad.png)
![screenshot from 2017-09-11 09-42-37](https://user-images.githubusercontent.com/6699411/30263654-1f372e7a-96d6-11e7-8721-2c79046c5c40.png)
![screenshot from 2017-09-11 09-42-58](https://user-images.githubusercontent.com/6699411/30263653-1f30a528-96d6-11e7-83d5-cc78907b7e25.png)
![screenshot from 2017-09-11 09-43-21](https://user-images.githubusercontent.com/6699411/30263656-1f5632b6-96d6-11e7-9310-bb8f92bf2a15.png)

so what's missing to be able to detect blocks of text ?

**Comment by argman** [comment]
@rmmal , thanks for trying EAST, this is interesting, can you provide a sample annotation of your data ?

In #18 , the result seems nice.

**Comment by rmmal** [comment]
You are welcome @argman  , Great effort btw.

As i said before i had a x1,y1 and x2,y2
where i converted them to x1,y1, x2,y1, x2,y2, x1,y2

![screenshot from 2017-09-11 13-00-31](https://user-images.githubusercontent.com/6699411/30271648-604f4756-96f1-11e7-9a92-febf5bde547f.png)
![screenshot from 2017-09-11 13-00-44](https://user-images.githubusercontent.com/6699411/30271646-5fe0a788-96f1-11e7-8647-605b035e1131.png)
![screenshot from 2017-09-11 13-00-59](https://user-images.githubusercontent.com/6699411/30271649-60520a18-96f1-11e7-9db0-ef494f27879b.png)
![screenshot from 2017-09-11 13-01-08](https://user-images.githubusercontent.com/6699411/30271644-5f942b7e-96f1-11e7-86f0-8f6acf01293c.png)
![screenshot from 2017-09-11 13-01-22](https://user-images.githubusercontent.com/6699411/30271645-5f945b4e-96f1-11e7-8a7c-7dc48d010a49.png)
![screenshot from 2017-09-11 13-01-31](https://user-images.githubusercontent.com/6699411/30271647-5fff47d8-96f1-11e7-8b73-f7992a274061.png)

so for example the annotations of the last image is 

**550,1667,833,1667,833,1737,550,1737,text
213,1679,493,1679,493,1731,213,1731,text
230,1741,802,1741,802,1777,230,1777,text
357,1330,687,1330,687,1458,357,1458,text
253,1492,790,1492,790,1539,253,1539,text
159,984,864,984,864,1093,159,1093,text**

what could be wrong ? suggestions ?

**Comment by rmmal** [comment]
setting text-scale 1024 , and input size 512 could make such a problem ?

**Comment by argman** [comment]
@rmmal , I think your problem is different. The definition of text maybe confusing for the network to learn. In the paper, text is defined as a single line. From your annotation, i cannot find what's the standard of a text line.

**Comment by rmmal** [comment]
I edited the dataset and started now training again , I will see what will happen and tell you.

Thanks for your help @argman

**Comment by MrOCR** [comment]
@rmmal any updates regarding your training & results?
